codec/src/main/java/io/netty/handler/codec/DefaultTextHeaders.java

```diff
@@ -579,7 +428,8 @@ private TextHeaders addEscapedValue(CharSequence name, CharSequence escapedValue
         }
 
         private <T> CharSequence commaSeparate(CsvValueEscaper<T> escaper, T... values) {
-            StringBuilder sb = new StringBuilder(values.length * DEFAULT_VALUE_SIZE);
```

Comment:
Why was `DEFAULT_VALUE_SIZE` removed?  Seems like we are reducing the visibility of this "configuration" type variable.